relay-server/src/envelope.rs

```diff
@@ -100,12 +100,14 @@ pub enum ItemType {
     MetricBuckets,
     /// Client internal report (eg: outcomes).
     ClientReport,
-    /// Profile event payload encoded in JSON
+    /// Profile event payload encoded as JSON.
     Profile,
-    /// Replay metadata and breadcrumb payload
+    /// Replay metadata and breadcrumb payload.
     ReplayEvent,
-    /// Replay Recording data
+    /// Replay Recording data.
     ReplayRecording,
+    /// Cron monitor checkin encoded as JSON.
+    Checkin,
```

Comment:
`Mon
```suggestion
    MonitorCheckin,
```

would maybe be consistent with what we have above, we don't call a replay recording `Recording` we call it a `ReplayRecording`.

Reply:
I'm usually preferring the shorter version, `ReplayRecording` was sort of an oversight.